zppy/e3sm_diags_vs_model.py

```diff
@@ -52,10 +52,11 @@ def e3sm_diags_vs_model(config, scriptDir):
             # List of dependencies
             dependencies = [ os.path.join(scriptDir, 'climo_%s_%04d-%04d.status' % (sub,c['year1'],c['year2'])), ]
 
-            # Submit job
-            jobid = submitScript(scriptFile, dependFiles=dependencies, export='NONE')
+            if not c['dry_run']:
```

Comment:
This file is never tested with `active = True`.

Reply:
Maybe this could be done later.